resources/views/competitions/partials/_form_competitions.blade.php

```diff
@@ -1,23 +1,18 @@
 @include('layouts.errors')
 
 <div class="form-item -padded">
-    {!! Form::label('campaign_id', 'Campaign ID:', ['class' => 'field-label']) !!}
-    {!! Form::text('campaign_id', (isset($room->campaign_id)) ? $room->campaign_id : NULL, ['class' => 'text-field']) !!}
+    {!! Form::label('contest_id', 'Contest ID:', ['class' => 'field-label']) !!}
+    {!! Form::text('contest_id', (isset($competition->contest_id)) ? $competition->contest_id : NULL, ['class' => 'text-field']) !!}
```

Comment:
I mentioned this in @deadlybutter PR too, but can you update this to use:

```php
{{ $competition->contest_id or old('contest_id') }}
```
It'll help with repopulating the field if there was a validation error with what the user entered and also feels much cleaner :) It's a little useful syntactic sugar :)